src/main/java/com/mongodb/hibernate/jdbc/MongoConnectionProvider.java

```diff
@@ -73,13 +77,16 @@
  * @see JdbcSettings#JAKARTA_JDBC_PASSWORD
  * @see <a href="https://www.mongodb.com/docs/manual/reference/connection-string/">connection string</a>
  */
-public final class MongoConnectionProvider implements ConnectionProvider, Configurable, Stoppable {
+public final class MongoConnectionProvider
```

Comment:
Why is this reformatted?  I thought introducing Spotless/Palantir would avoid these sorts of changes.

Reply:
It is because a new interface was added to the `implements` list, ending up with the line exceeding 120 chars real estate, so Spotless/Palantir has to break it down into two lines.

Reply:
oh, totally missed that!  Sorry.